src/components/common/BoardSection.tsx

```diff
@@ -21,20 +21,20 @@ const ICON = {
   콘텐츠: <FolderIcon fill="#292929" active />,
 };
 
-export default function BoardSection({ title, content, children }: BoardSection) {
+export default function BoardSection({ title, content, mode }: BoardSection) {
```

Comment:
```suggestion
export default function BoardSection({ title, mode, content, children}: BoardSection) {
```
기존 작업 부분이 삭제돼서 이렇게 바꿔주세요!